docs/02-architecture/02-fronts-architecture.md

```diff
@@ -1,69 +1,110 @@
+In a nutshell:
+- [Facia press](#facia-press): for each front, fetch all necessary data, aggregate it and store it in S3
+- [Facia](#facia): read the aggregated data from S3 and render the front
 
-The global list of defined fronts - and the definitions of the collections on those fronts - is held in a single configuration file (`config.json`). This file is edited using the Configuration Tool. It is polled and referred to by each component in the architecture. 
+## Facia Press
+It is not a user facing service, it processes events published to queues.
+There are 2 queues:
+- one queue populated by a cron job running in the Admin app.
+- one queue where the Fronts tool (fronts.gutools.co.uk) publishes events when a front is being updated.
+The cron job is very important since content for a front can be coming from CAPI and change without any update from the Fronts tool.
 
-The running order of articles in each collection - and various tweaks to those articles - are represented in multiple `collection.json` files. These are individually edited using the Fronts Tool and persisted to S3. These `collection.json` files dictate which articles need to be requested from the ContentApi during the production of each front's `pressed.json` files.
+### Pressing a front:
+In reaction to a new event published to the queues described above, Facia-press will:
 
-As soon as a collection is edited, its id is added to a queue consumed the Presser, which will re-create a `pressed.json` file for any front containing that collection. Additionally, each front is periodically queued for re-pressing by the Admin app; fronts have differential rates of periodic re-pressing, to reflect their differing priorities.
+1. Get the collections that compose the front by fetching the Fronts config at `s3://facia-tool-store/STAGE/frontsapi/config/config.json` in the CMS Fronts account. This file is edited using the Configuration Tool
 
-The `pressed.json` files - one for each front - are persisted to S3. Both preview and live versions are produced.
+Example of a front config:
+```
+...
+"uk/business": {
+    "collections": [
+        "uk/business/regular-stories",
+        "uk/business/feature-stories",
+        "12554364-5a48-4e1a-a931-4dbcd144a12a",
+        "98257c5b-f7af-4238-994e-edfb17e3b16e",
+        "7aa73028-4e76-48d7-a44e-b62a15d36a65",
+        "uk/business/most-viewed/regular-stories",
+        "b5058768-e4a8-4e37-a61e-5f8e4df7f8fc"
+    ],
+    "webTitle": "Economy",
+    "title": "Latest financial, market & economic news and analysis | Business",
+    "description": "Latest financial, market &amp; economic news and analysis from The Guardian"
+}
+...
+```
```

Comment:
Might be useful to distinguish between behaviour of draft and live presses?  Eg. behaviour when front is not yet published and the pressing schedule.